orchestra/static/dist/main.js

```diff
@@ -47608,7 +47608,6 @@ function orchestraTasks($http) {
     data: null,
```

Comment:
all changes to javascript here are to remove the preventNew/get new task functionality in the old Angular version of the experience.